ResourceStatusSystem/Client/SiteStatus.py

```diff
@@ -179,7 +179,27 @@ def getUsableSites( self, siteNamesList ):
     """
 
     if not siteNamesList:
-      return S_ERROR(DErrno.ERESUNK, 'siteNamesList is empty')
+      if self.rssFlag:
+        activeSites = self.rsClient.selectStatusElement( 'Site', 'Status', status = 'Active', meta = { 'columns' : [ 'Name' ] } )
+        if not activeSites['OK']:
+          return activeSites
+
+        degradedSites = self.rsClient.selectStatusElement( 'Site', 'Status', status = 'Degraded', meta = { 'columns' : [ 'Name' ] } )
+        if not degradedSites['OK']:
+          return degradedSites
+
+        return activeSites['Value'] + degradedSites['Value']
```

Comment:
returning here seems a mistake to me, all code after https://github.com/vipersec/DIRAC/blob/90294f3bdba860c5d41988a2c7f7842983c6ad14/ResourceStatusSystem/Client/SiteStatus.py#L206 becomes unreachable.